zsh-kubectl.plugin.zsh

```diff
@@ -17,3 +17,8 @@ alias k=kubectl
 # Add completions to the FPATH
 typeset -TUx FPATH fpath
 fpath=("$COMPLETIONS_DIR" $fpath)
+
+# Make it easy to reset the completions file when a new kubectl version is released
```

Comment:
hm i'm not sure if this is the best way to go here.. Don't you think it would be better to automatically update it instead of manually trigger this? I thought about something like this in line 10 (this is also implemented in [this](https://github.com/wintermi/zsh-rust) zsh plugin): 
```zsh
if [[ ! -f "$COMPLETIONS_DIR/_kubectl"  ||  ! $(find "~/.local/share/zap/plugins/zsh-kubectl/completions/_kubectl" -newermt "24 hours ago" -print) ]]; then !
```

The downside of my approach would be the huge input lag every time the `kubectl completion zsh` is triggered.. what do you thin?

Reply:
I'm hesitant to add anything that does a check every new shell session because every one of those adds a delay to new sessions.

Maybe do the age check & deletion in a function and run it in the background. Then you'll get a quick session start, but if the completions are stale, the next session start will regenerate them.

Reply:
and what about lazy loading the whole `kubectl` function like [this](https://frederic-hemberger.de/notes/shell/speed-up-initial-zsh-startup-with-lazy-loading/)

Then we only have the small delay if we execute the command but not after creating a new session

Reply:
That's neat - I'll update the PR to use that instead. I have a bunch of other plugins that would benefit from that approach too.

Thanks for the tip!